gammapy/irf/psf_analytical.py

```diff
@@ -354,3 +358,46 @@ def info(self, fractions=[0.68, 0.95], energies=Quantity([1., 10.], 'TeV'),
                            "E = {2:4.1f}: {3:5.8f}\n"
                            "".format(100 * fraction, theta, energy, radius))
         return ss
+
+
+
+    def to_table_psf(self, theta=None, offset_max=None, binsz=0.01, exposure=None):
+        """
+        Convert triple Gaussian PSF ot table PSF.
+
+        Parameters
+        ----------
+        theta : `~astropy.coordinates.Angle`
+            Observation offset. Default theta = 0 deg.
+        offset_max : `~astropy.coordinates.Angle`
+            Maximum offset from PSF center. Default offset_max = 2.5 deg
+        binsz : float
+            Size of the spatial bins in deg / pixel. Default binsz = 0.01 deg
+        exposure : `~astropy.units.Quantity`
+            Energy dependent exposure. Should be in units equivalent to 'cm^2 s'.
+
+        Returns
+        -------
+        tabe_psf : `~gammapy.irf.EnergyDependentTablePSF`
+            Instance of `EnergyDependentTablePSF`.
+        """
+        # Convert energies to log center
+        ebounds = EnergyBounds.from_lower_and_upper_bounds(self.energy_lo, self.energy_hi)
+        energies = ebounds.log_centers
+        
+        # Default for exposure
+        exposure = exposure or Quantity(np.ones(len(energies)), 'cm^2 s')
+        theta = theta or Angle(0, 'deg')
+        offset_max = offset_max or Angle(2.5, 'deg')
+
+        offsets = Angle(np.arange(0, offset_max.degree, binsz), 'deg')
+
+        psf_value = Quantity(np.empty((len(energies), len(offsets))), 'deg^-2')
+
+        for i, energy in enumerate(energies):
+            psf_gauss = self.psf_at_energy_and_theta(energy, theta)
+            psf_value[i] = Quantity(psf_gauss(offsets, np.zeros_like(offsets)), 'deg^-2')
```

Comment:
What Gammalib is doing is interpolating the PSF parameters.
We could do the same.
But I'm not sure this is a good, maintainable way to go, so I would suggest to leave what we have now (you could call it no or nearest neighbor interpolation) for analytical IRFs.

Generally I'm not convinced we need analytical IRFs at DL3 at all.
I think IRF3 producers should just use good binnings and always histogram their IRFs before handing them out (if they had analytical IRFs internally), and then it's really simple for science tools.

So in Gammapy for now we do want to support analytical PSFs, because people use them, but my suggestion to always convert to histogram PSF immediately and then to only develop and test that class well.

So @JouvinLea - We need the equivalent `to_table_psf` for the King PSF in Gammapy so that we can analyse Model++ data. Would you be willing to implement it in the coming days (after the review on this one is done and you have a good template how it should be implemented)?